src/instructlab/model/train.py

```diff
@@ -87,29 +165,67 @@
         "(reduces GPU VRAM usage and may slow down training)"
     ),
 )
+@click.option("--max-seq-len", type=int)
+@click.option("--max-batch-len", type=int)
 @click.option(
-    "--model-name",
-    default="instructlab/merlinite-7b-lab",
-    show_default=True,
-    help="model name to use in training",
+    "--effective-batch-size", type=int, help="total batch size across all GPUs"
+)
+@click.option(
+    "--save-samples",
+    type=int,
+    help="how many amples to process before saving a checkpoint",
+)
+@click.option("--learning-rate", type=float, help="learning rate for training")
+@click.option("--warmup-steps", type=int, help="warmup steps for training")
+@click.option(
+    "--cpu-offload-optimizer", type=bool, help="if true enables optimizer offload"
+)
+@click.option("--cpu-offload-ratio", type=float, help="cpu offload optimizer ratio")
+@click.option(
+    "--cpu-offload-optimizer-pin-memory",
+    type=bool,
+    help="if true pin memory when using cpu optimizer",
+)
+# below flags are invalid if lora == false
+@click.option("--rank", type=int, help="rank of update matricies")
+@click.option("--alpha", type=float, help="how influential/strong lors tune will be")
+@click.option("--dropout", type=float, help="dropout for LoRA layers")
+@click.option(
+    "--target-modules",
+    type=str,
 )
+@click.option("--quantize-data-type", type=str, default=None)
+@click.option(
+    "--is-padding-free",
+    type=bool,
+)
+@click.option(
+    "--gpus",
+    "nproc_per_node",
+    type=int,
+    help="this is the number of GPUs to use. This is a torch specific arg and must be called nproc-per-node",
+)
+@click.option("--nnodes", type=int)
+@click.option("--node-rank", type=int)
+@click.option("--rdzv-id", type=int)
+@click.option("--rdzv-endpoint", type=str)
```

Comment:
This is the rendezvous endpoint which other torchrun jobs will join on. Usually in the format of `url`, can be `<ip>:<port>`, `<dns-name>`, etc.